unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs

```diff
@@ -57,6 +57,7 @@ public class PlacesSubSectionComponentController : IPlacesSubSectionComponentCon
         this.view.OnJumpInClicked += JumpInToPlace;
         this.view.OnFriendHandlerAdded += View_OnFriendHandlerAdded;
         this.view.OnShowMorePlacesClicked += ShowMorePlaces;
+        DataStore.i.channels.currentJoinChannelModal.OnChange += OnChannelToJoinChanged;
```

Comment:
you should inject the datastore instead of using the singleton in this class

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2724/commits/94abfca70f28dbd712029cfdeed0fa1119e44b4a